domains/*.nura.is-not-a.dev.json

```diff
@@ -0,0 +1,11 @@
+{
+    "description": "Wildcard CNAME Records",
+    "domain": "is-not-a.dev",
+    "subdomain": "*.nura",
+    "record": {
+        "CNAME": ["key1.nura.is-not-a.dev._domainkey.migadu.com."],
+        "CNAME": ["key2.nura.is-not-a.dev._domainkey.migadu.com."],
+        "CNAME": ["key3.nura.is-not-a.dev._domainkey.migadu.com."]
+    },
```

Comment:
Invalid CNAMEs, you can only have one, must be one string, no object. See DNS rules for more information.